### DISCUSSION TITLE
TTS does not pronounce the numbers

### DISCUSSION THREAD

**Discussion by SonosInc** [snakers4/silero-models#221] [Q&A] [answered]
Hello! TTS does not pronounce the numbers on the ru_v3 model, it simply skips. Are there any problems with this?
Thank you!

**Comment by snakers4** [comment]
There is no text normalization module built-in.

**Comment by S-trace** [comment, marked as answer]
Try my https://github.com/S-trace/silero_tts_standalone script - it contains some basic text normalization including numbers-to-text conversion for Russian language with https://github.com/seriyps/ru_number_to_text
You can use any number-to-text module for any language with this script - just change spell_digits() function.
